### PR TITLE
feat: V3 Airdrop

### PR DESCRIPTION
**Pull request by ChefMomota**

### CODE REVIEW

File: apps/web/src/components/GlobalCheckClaimStatus/V3AirdropModal.tsx [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+  const textDisplay = (): string => {
+    if (hasPart1 && hasPart2) {
+      return t(
+        "You've earned an exclusive v3 Early Supporter and v3 Legendary NFT for participating in the PancakeSwap v3 Launch campaign. Claim your NFT now and show off your support for PancakeSwap.",
+      )
+    }
+    if (hasPart1) {
+      return t(
+        "You've earned an exclusive v3 Lgendary NFT for participating in the PancakeSwap v3 Launch campaign. Claim your NFT now and show off your support for PancakeSwap.",
```
Comment (Aosiepcs) [inline apps/web/src/components/GlobalCheckClaimStatus/V3AirdropModal.tsx:82]: typo - Legendary*

File: apps/web/src/components/GlobalCheckClaimStatus/index.tsx [hunk 2]
```diff
@@ -31,48 +32,79 @@ const GlobalCheckClaimStatus: React.FC<React.PropsWithChildren<GlobalCheckClaimS
...
+      } else {
+        closeV3AirdropModal()
+      }
     }
 
     if (account) {
       fetchClaimAnniversaryStatus()
     }
-  }, [account, canClaim])
+  }, [data, account, canClaimReward, isClaimed, closeV3AirdropModal])
```
Comment (chef-jojo) [inline apps/web/src/components/GlobalCheckClaimStatus/index.tsx:97]: only on bsc

Comment (ChefMomota) [reply apps/web/src/components/GlobalCheckClaimStatus/index.tsx]: done

File: apps/web/src/components/GlobalCheckClaimStatus/index.tsx [hunk 3]
```diff
@@ -31,48 +33,80 @@ const GlobalCheckClaimStatus: React.FC<React.PropsWithChildren<GlobalCheckClaimS
...
+    return {
+      v3WhitelistAddress,
+      v3ForSC,
+      v3MerkleProofs,
+    }
+  })
+
+  const [onPresentV3AirdropModal, closeV3AirdropModal] = useModal(
+    <V3AirdropModal
+      data={data?.v3WhitelistAddress[account] as WhitelistType}
```
Comment (chef-jojo) [inline apps/web/src/components/GlobalCheckClaimStatus/index.tsx:71]: we have to use lowercase if you use github as static file

Comment (ChefMomota) [reply apps/web/src/components/GlobalCheckClaimStatus/index.tsx]: Fixed.Nice catch

**Review by chef-jojo — APPROVED** [review]

**Pull request merged**